src/Compilers/CSharp/Portable/BoundTree/IBoundInvocableExpression.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    internal interface IBoundInvocableExpression
+    {
+        BoundExpression GetAsBoundNode();
```

Comment:
>BoundExpression GetAsBoundNode(); [](start = 8, length = 33)

Consider removing this member, it looks like all implementers return itself, a simple cast would work.